applications/incompressible_navier_stokes/flow_past_sphere/application.h

```diff
@@ -135,9 +135,10 @@ class Application : public ApplicationBase<dim, Number>
     this->param.rel_tol_steady = 1.e-8;
 
     // SPATIAL DISCRETIZATION
-    this->param.grid.triangulation_type = TriangulationType::Distributed;
-    this->param.grid.mapping_degree     = this->param.degree_u;
-    this->param.degree_p                = DegreePressure::MixedOrder;
+    this->param.grid.triangulation_type           = TriangulationType::Distributed;
+    this->param.grid.mapping_degree               = this->param.degree_u;
+    this->param.degree_p                          = DegreePressure::MixedOrder;
+    this->param.grid.create_coarse_triangulations = true;
```

Comment:
Please add a comment like "we need to create coarse triangulations since we use global coarsening multigrid" in order to help other users in understanding the reason behind this setting